err_max : 0.15
if_debug : 1

## for colision_avoidance
##目标位置
target_x_mission_2 : 8
target_y_mission_2 : -2
target_x_mission_3 : 15
target_y_mission_3 : 0.5
target_yaw : 0

##追踪部分位置环P
p_xy : 1.4

## 单位:米
R_outside : 1.0

## 单位:米
R_inside : 0.6

## 追踪部分速度限幅
vel_track_max : 0.25

## 大圈比例系数
p_R : 0.6

## 小圈比例系数
p_r : 1.2

## 避障部分最大速度
vel_collision_max : 0.30

## 总速度最大速度
vel_sp_max : 0.3


## 激光雷达范围设定
range_min : 0
range_max : 359
